test/parallel/test-debug-port-from-cmdline.js

```diff
@@ -38,11 +38,12 @@ function processStderrLine(line) {
 
 function assertOutputLines() {
   const expectedLines = [
-    'Starting debugger agent.',
-    'Debugger listening on 127.0.0.1:' + debugPort,
+    /Starting debugger agent./,
+    /\(node:\d+\) DeprecationWarning:.*/,
```

Comment:
I looked into `common.expectWarning()` but it won't be possible to use here since we're checking stderr from a child process, so left logic as is.

Added more of the deprecation warning to the regex.